### DISCUSSION TITLE
ESMValTool community meeting June 2024 - Pacific Region Friendly

### DISCUSSION THREAD

**Discussion by rbeucher** [ESMValGroup/Community#166] [Meetings]
Hello ESMValTool Community,

Recognizing the diversity of our community and the challenges posed by different time zones, we are organising a community meeting at a time suitable for the Pacific region.

The next online community meeting will be on **June 14th at 0930AM AEST, 0430PM PDT**

The duration will be one hour.

Calendar invitations and Zoom links to the meeting are sent to the mailing list, join it [here.](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list)

## Format and agenda

Anyone with an interest in ESMValTool is welcome to join and ask questions

We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.

Using the input from the introduction round, we choose some topics to discuss in more detail.

If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find the other people in the meeting and we can start discussing there before the meeting. Create a new GitHub issue about your topic if there is no open issue yet.

GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

**Comment by k-a-webb** [comment]
Hi, I would appreciate a discussion on the changes to the regrid preprocessor since v2.5, related to this issue:
[Longer run time of recipes with 3D regridding in ESMValTool v2.10.0 compared to v2.5.0 #3590](https://github.com/ESMValGroup/ESMValTool/issues/3590).

**Reply by k-a-webb** [reply]
^ This issue has now been resolved, but it would be helpful to have a discussion about lazy/non-lazy data and which preprocessors can use both.

**Reply by bouweandela** [reply]
Almost all preprocessor functions support realized data, see https://github.com/ESMValGroup/ESMValCore/issues/674 for an overview of which preprocessors support lazy data.

**Comment by flicj191** [comment]
updates:
@flicj191 - will work on using the python api in a notebook and making some training material
@rhaegar325 - working on live-CMORiser for ACCESS models PR ESMValGroup/ESMValCore#2430
@rbeucher - working on ERA5 daily collection at NCI(Australia infrastructure) for integration and use in esmvaltool, ACCESS recipes for hackathon training event in September, optimising dask cluster on Gadi (supercomputer)
@hot007 - question around regional performance metrics, and CORDEX support

Thanks all, feel free to update/edit
